src/Websites/Platform/src/Bit.Websites.Platform.Client/Pages/Home/HomePage.razor.scss

```
@@ -1,5 +1,6 @@
 ﻿@import '../../Styles/abstracts/_vars.scss';
 @import '../../Styles/abstracts/_colors.scss';
+@import '../../Styles/abstracts/_mixins.scss';
```

Stylelint (17.14.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)